README.md

```diff
@@ -54,6 +54,7 @@ Please read [CONTRIBUTING](./CONTRIBUTING.md) if you wish to add a new vector cl
 *Classes that wrap complex objects.*
 
 * [`blob`](https://blob.tidyverse.org/reference/blob.html) - Binary Large OBjects: [`blob[42 B]`](reprex/blob.md)
+* [`labelled`](https://haven.tidyverse.org/reference/labelled.html) - Labelled vectors: [`labelled(1, c("label of value 1" = 1))`](reprex/labelled.md)
```

Comment:
The overview is meant to contain examples on how the vector prints inline. This is complicated here because `"labelled"` doesn't print in one line. I'll update CONTRIBUTING.

Maybe:

```suggestion
* [`labelled`](https://haven.tidyverse.org/reference/labelled.html) - Labelled vectors: [`<Labelled double> 3` with labels](reprex/labelled.md)
```